medmodels/treatment_effect/continuous_estimators.py

```diff
@@ -39,9 +39,9 @@ def average_treatment_effect(
     Args:
         medrecord (MedRecord): An instance of the MedRecord class containing medical
             data.
-        treatment_true_set (Set[NodeIndex]): A set of node indices representing the
+        treatment_outcome_true_set (Set[NodeIndex]): A set of node indices representing the
             treated group.
```

Comment:
Isnt that the set that represents nodes that are true for outcome and treatment? Therefore the sentence _"A set of node indices representing the treated group."_ would be incorrect?